public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -3,24 +3,18 @@ import 'dart:math' as math;
 import 'package:angular2/angular2.dart';
 
 /*
-* Raise the value exponentially
-* Takes a value that defaults to 0 and an exponent argument that defaults to 1.
-* Checks for value to be a string or number.
-* Usage:
-*   value | exponentialStrength:exponent
-* Example:
-*   {{ 2 |  exponentialStrength:10}}
-*   formats to: 1024
-*/
-
+ * Raise the value exponentially
+ * Takes an exponent argument that defaults to 1.
+ * Usage:
+ *   value | exponentialStrength:exponent
+ * Example:
+ *   {{ 2 |  exponentialStrength:10}}
+ *   formats to: 1024
+ */
 @Pipe(name: 'exponentialStrength')
-@Injectable()
+@Injectable() // FIXME(chalin): unnecessary?
```

Comment:
That is what I thought, but I wanted to double check in case there was a nonstandard case where it might have been necessary. Thanks for confirming. @thso do you agree?